### ISSUE TITLE
[Question/Feedback]: Enable team linter and other bots to handle utility modules

### ISSUE THREAD

**Issue by matebarabas** [Azure/Azure-Verified-Modules#1032]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Description

Let's incorporate the new utility module class in all the custom automation solutions!

**Comment by matebarabas** [comment]
@ChrisSidebotham, yes please, we need the functionality; I don't know however whether it has been already implemented and it's just this issue was left open. All this is about is to make sure that issues related to utility modules are also handled the same way as the ones for res or ptn modules. Please take a quick look at our scripts and check that we didn't leave out utl modules. Thanks!

**Comment by ChrisSidebotham** [comment]
closing as we don't use this utility for team amangement anymore
